backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -487,12 +487,12 @@ public static String convertTestResultToSnomed(TestResult result) {
     return concept.code();
   }
 
-  public static String convertConceptCodeToConceptName(String snomedCode) {
+  public static SnomedConceptRecord convertConceptCodeToConceptName(String snomedCode) {
```

Comment:
can this method be renamed?

also is it okay that other conversion "sister" methods above this are still using INVALID for their default behavior?

Reply:
The sister methods are not used by the bulk upload flow as far as I can tell, which is why I'm not rewriting all of them.  The `else` is still incorrect, but I don't think it is a reachable code block based on our frontends behavior.

Trying to keep the scope of this initial PR small, because there's a lot of rabbit trails we could follow here.